src/ComponentLibraryManagerAddOn/ComponentLibraryManagerAddOn.cs

```diff
@@ -148,6 +149,71 @@ public void ObjectEvent(MgaObject subject, uint eventMask, object param)
                     Process(CyPhyClasses.ComponentAssembly.Cast(subject));
                 }
             }
+            else if ((eventMask & (uint)objectevent_enum.OBJEVENT_ATTR) != 0 &&

```

Comment:
I'd like to see a comment or alias for these expressions. For example, above we have `objectCreatedAndNotLibObject`.